test/basic.html

```diff
@@ -721,6 +740,73 @@
       });
     });
 
+    suite('content type', function() {
+        var server;
+        var f;
+
+        setup(function() {
+            server = sinon.fakeServer.create();
+            f = fixture('content-type');
+        });
+
+        teardown(function() {
+            server.restore();
+        });
+
+        test('submits a form', function(done) {
```

Comment:
This indentation seems whack, but I'll merge this PR and fix it myself, so that I don't keep you waiting :)